docs/pages/Cardinal/Cross_Shard_Communication/evm_to_cardinal.mdx

```diff
@@ -0,0 +1,188 @@
+import { Callout } from 'nextra/components'
+
+<Callout type={"warning"}>
+    EVM to Cardinal communication is an experimental feature. The API is not final and bugs are expected to be encountered.
+</Callout>
+
+
+# EVM To Cardinal Communication
```

Comment:
I don't think the documentation is read from start to finish. So many users come in here without fully understanding cardinal as well. This section of the documentation should reference how cardinal does this without the evm. Some slight background on that plus a link then it should emphasize that this section of the docs is identical to that but from the evm side.

Reply:
can you expand on "how cardinal does this without the evm"?

Reply:
It's just messages/transactions right? So cardinal allows messages to come in via an http route but it can be done with smart contracts too. So I would have the documentation say this, that this is an alternative way to the http method.

Reply:
ahh got it.

Reply:
[add aside that cardinal normally gets messages through game relayer](https://github.com/Argus-Labs/world-engine/pull/468/commits/e50175742c4331328dd48a40bb11eabcd3afb02d)

hows this?